### PR TITLE
[14.0][ADD] account_fiscal_year: forward port #861

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**
Forward port https://github.com/OCA/account-financial-tools/pull/861

### CODE REVIEW

File: account_fiscal_year/__manifest__.py [hunk 1]
```diff
@@ -21,5 +21,6 @@
         "security/ir.model.access.csv",
         "security/account_fiscal_year_rule.xml",
         "views/account_fiscal_year_views.xml",
+        "data/date_range_type.xml",
```
Comment (AaronHForgeFlow) [inline account_fiscal_year/__manifest__.py:24]: Sorry for being picky but I think this should come before views declaration

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM!

**Comment by MiquelRForgeFlow** [comment]
Should be closed if superseded by https://github.com/OCA/account-financial-tools/pull/1265.

**Comment by dreispt** [comment]
Superseded by #1265

**Pull request closed without merging**